react/yarn.lock

```diff
@@ -4867,9 +4867,9 @@ verror@1.10.0:
   version "0.7.1"
   resolved "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.native-types@0.7.1/public/@types/vtex.native-types#a197484a2fdcee5c61a6d20ad6c994faa58380f5"
 
-"vtex.product-context@https://victormiranda4--storecomponents.myvtex.com/_v/private/typings/linked/v1/vtex.product-context@0.8.1+build1599137148/public/@types/vtex.product-context":
+"vtex.product-context@https://sellername--storecomponents.myvtex.com/_v/private/typings/linked/v1/vtex.product-context@0.8.1+build1599136488/public/@types/vtex.product-context":
```

Comment:
Reminder to update this once https://github.com/vtex-apps/product-context/pull/32 is released.